src/application.ts

```diff
@@ -150,20 +156,19 @@ export default class Application {
 
     Object.keys(data.relationships).forEach(relationshipName => {
       if (Array.isArray(data.relationships[relationshipName])) {
-        data.relationships[relationshipName] = (data.relationships[
-          relationshipName
-        ] as any).map(rel => {
-          rel["type"] = schemaRelationships[relationshipName].type().type;
-          return rel;
-        });
+        data.relationships[relationshipName] =
+          (data.relationships[relationshipName] as any).map(rel => {
```

Comment:
@spersico Can you add a `// TODO` here so we can take care of this `any` later? (It's probably mine, but just so I don't forget about it.)

Reply:
will do